### DISCUSSION TITLE
Can not load the audio （file in the local）

### DISCUSSION THREAD

**Discussion by yg2342423523** [katspaugh/wavesurfer.js#2882] [Q&A]
sir，one question，i use the url to load my wave，it works！
```
var wavesurfer = WaveSurfer.create({
		container: test2,
		waveColor: '#0091ff',
		progressColor: '#ccc',
		height: 50,
		barMinHeight: 1,
		barWidth: 2,
		// barGap: 3,
		// barRadius: 11,
		fillParent: true,
		cursorWidth: 0,
		hideScrollbar: true,
	});
	wavesurfer.load(url, readPeaks(name));
```

url is this：["D:/packs/啵 14 (liquid).wav"]

but when i add methods to it，it will be wrong；
![dc6e072eff11c54d759abcaa8f029ad](https://github.com/katspaugh/wavesurfer.js/assets/102865541/9f1250a4-954b-4666-8e3e-f33ae8963a3e)

```
	wavesurfer.once('interaction', () => {
		wavesurfer.play()
	})
```

i think it is still the problem of blob，right？

I DON'T know how to make this

**Comment by katspaugh** [comment]
As the error says, it cannot load audio via a file:// URL. You need to run an HTTP server and load it from there via http:// or https://.
